### DISCUSSION TITLE
Caching for isomorphic-git on mobile?

### DISCUSSION THREAD

**Discussion by mihail-yartsev** [Vinzent03/obsidian-git#328] [General]
Hi, 
I see there's finally support for git on mobile, which is fantastic! (I waited at least a year for it)
The description says there are some performance problems with the status command.
When I google it I see a caching solution to solve the problem that isomorphic-git guys implemented: https://isomorphic-git.org/docs/en/cache
I tried checking the code, but I'm not very good with JS so I didn't understand if obsidian git is using it already 😁

It can be cool if someone could help me with this question. I really would like to use git on mobile conviniently.

Sorry if this was already discussed elsewhere, I tried searching issues and discussions and didn't find anything.

**Comment by Vinzent03** [comment]
In my understanding, it only helps for calling many single methods, but I'm already using the suggested `statusMatrix` method, which I think does this internally.
In addition they say:
> For memory constrained devices, you may want to not use a cache at all.

So it may cause memory issues on mobile.
